schema/release-schema.json

```
@@ -2283,6 +2293,26 @@
           ]
         }
       }
+    },
+    "Link": {
+      "title": "Link",
+      "description": "A typed connection to a related resource.",
+      "type": "object",
+      "properties": {
+        "rel": {
+          "title": "Link Relation Type",
+          "description": "The semantics of the link, using the open [link relation type](https://standard.open-contracting.org/{{version}}/{{lang}}/schema/codelists/#link-relation-type) codelist.",
+          "type": "string",
+          "codelist": "linkRelationType.csv",
+          "openCodelist": true
+        },
+        "href": {
+          "title": "Link Target",
+          "description": "The target URI of the link.",
```

Use URL instead of URI.

```suggestion
          "description": "The URL of the related resource.",
```